lib/snap_deploy/provider/aws/ops_works.rb

```diff
@@ -24,7 +25,7 @@ def execute
   def create_deployment
     data = client.create_deployment(
       stack_id: ops_works_app[:stack_id],
-      app_id:   app_id,
+      **deploy_target,
```

Comment:
Another nitpick - could this be changed to a simple merge instead of the double splat?